src/llv8-inl.h

```diff
@@ -352,13 +352,29 @@ inline Value DescriptorArray::GetValue(int index, Error& err) {
 }
 
 inline bool DescriptorArray::IsFieldDetails(Smi details) {
-  return (details.GetValue() & v8()->descriptor_array()->kPropertyTypeMask) ==
-         v8()->descriptor_array()->kFieldType;
+  constants::DescriptorArray* d = v8()->descriptor_array();
+  int64_t v = details.GetValue();
+
```

Comment:
I know, but with longer names the code below gets line breaks at really awkward places because of the 80 columns rule.